tools/tox.ini

```diff
@@ -21,3 +21,8 @@ commands = flake8 --append-config={toxinidir}/py27-flake8.ini {posargs}
 [testenv:py36-flake8]
 deps = -rrequirements_flake8.txt
 commands = flake8 --append-config={toxinidir}/py36-flake8.ini {posargs}
+
+[testenv:mypy]
+basepython = python3
+deps = mypy
+commands = mypy --py2 --ignore-missing-imports manifest
```

Comment:
Can you add a `tools/mypy.ini` with the appropriate stuff in it, rather than adding ever more arguments here? Also wonder how much of `--strict` we can explicitly enable easily?